src/components/datepicker/datePicker.js

```diff
@@ -351,7 +516,12 @@
    */
   DatePickerCtrl.prototype.setDisabled = function(isDisabled) {
     this.isDisabled = isDisabled;
-    this.inputElement.disabled = isDisabled;
+    if (!this.isDateRangePicker) {
```

Comment:
Because depending on the context of the directive, this.inputElement or this.startInputElement/this.endInputElement will be null. This means the .disabled call on a null object will be undefined.